indico/modules/events/abstracts/templates/management/abstracts_setup_actions/ended.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'events/abstracts/management/abstracts_setup_actions/_base.html' %}
```

Comment:
Although I don't like using acronyms, I think your proposal is better since it represents exactly what the folder is used for.